.gitignore

```diff
@@ -3,3 +3,4 @@
 target
 tmp
 .DS_Store
+.mvn
```

Comment:
What's up with putting this in .gitignore and removing the extension? It's necessary for the CI and our versioning approach

Reply:
well, it screws up my maven build (and IIRC you had issues with it too)...but it's ok, we can keep it and I will just delete it locally.